### PR TITLE
update tutorials for t.u.c launch

### PR DESCRIPTION
**Pull request by ghost**
don't review, work in progress

### CODE REVIEW

File: templates/pages/core/tutorials.md [hunk 1]
```diff
@@ -7,30 +7,24 @@ description: A series of tutorials, step by step practical guides to help you ac
 
 Tutorials are practical guides, with step-by-step instructions to achieve the most common development tasks. They cover a variety of areas from getting started tasks to more advanced topics. Easy to follow, practical, and instructive, they should give you an understanding of key Ubuntu Core concepts and, more importantly, help you build components of your solution.
 
-Tutorials are still in beta, but to get an early preview, you can access the offline snap version of the tutorials. To give us feedback or to ask for new tutorials to be created, please [file an issue](https://github.com/ubuntu/codelabs/issues) on the GitHub project.
+### [Ubuntu Core tutorials] (tutorials.ubuntu.com)
```
Comment (caldav) [inline templates/pages/core/tutorials.md]: For links to work, there should be no spaces between `](`.
Link should be absolute: https://tutorials.ubuntu.com

File: templates/pages/core/tutorials.md [hunk 2]
```diff
@@ -7,30 +7,24 @@ description: A series of tutorials, step by step practical guides to help you ac
...
-You can then access the snap by visiting `http://localhost:8123/` in your browser.
-
-## Available tutorials
+## Latest tutorials
 
 ### Create your first snap
 
 This tutorial covers using snapcraft. It will walk you through creating your first snap and learning the core snapcraft concepts.
 
+[Start now] (https://tutorials.ubuntu.com/tutorial/create-first-snap#0)
```
Comment (caldav) [inline templates/pages/core/tutorials.md]: There should be no space between `](`

File: templates/pages/core/tutorials.md [hunk 3]
```diff
@@ -7,30 +7,24 @@ description: A series of tutorials, step by step practical guides to help you ac
...
+
 ### Basic snap usage
 
 A tutorial about using snaps to distribute code and the main benefits of using snaps.
 
-### Building a nodejs service
+[Start now] (https://tutorials.ubuntu.com/tutorial/basic-snap-usage#0)
```
Comment (caldav) [inline templates/pages/core/tutorials.md]: There should be no space between `](`

File: templates/pages/core/tutorials.md [hunk 4]
```diff
@@ -7,30 +7,24 @@ description: A series of tutorials, step by step practical guides to help you ac
...
+
+### Building a nodejs service 
 
 Using the nodejs snapcraft plugin to build a simple service, covering good practice for debugging and iterating over your web server, and basic confinement notions.
+
+[Start now] (https://tutorials.ubuntu.com/tutorial/build-a-nodejs-service#0)
```
Comment (caldav) [inline templates/pages/core/tutorials.md]: There should be no space between `](`

**Review by caldav — CHANGES_REQUESTED** [review]
Some fixes required for links to work

**Comment by ghost** [comment]
@caldav fixed, please have a look

**Comment by caldav** [comment]
LGTM!

**Review by caldav — APPROVED** [review]

**Pull request merged**